lib/database/mysql.ex

```diff
@@ -61,9 +61,12 @@ defimpl Plsm.Database, for: Plsm.Database.MySql do
     {_,type} = start_type
     upcase = String.upcase type
     cond do 
-      String.starts_with?(upcase, "INT") == true -> :integer
-      String.starts_with?(upcase, "BIGINT") == true -> :integer
-      String.starts_with?(upcase, "TINYINT") == true -> :integer
+      String.contains?(upcase, "INT") == true -> :integer
```

Comment:
This will convert tinyint(1) into an integer when, in fact, it is mapped to :boolean